pkg/kubectl/cmd/util/factory_test.go

```diff
@@ -37,6 +37,7 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/sets"
```

Comment:
@xiangpengzhao  I think it may be related with China GFW. Some go packages can not be got.